test/junit/scala/sys/process/ProcessTest.scala

```diff
@@ -22,4 +22,9 @@ class ProcessTest {
     val res = ("cat" #< ( () => -1 ) ).!
     assertEquals(0, res)
   }
+
+  @Test def t10953(): Unit = testily {
```

Comment:
It looks like `testily` was used to exclude the other test on Windows, because it requires a `cat` command.

But just testing `Process.cat` should be OK on Windows, right?

Reply:
I suppose you are right but I will not be able to confirm that it works in Windows. Should I remove it and wait that someone tries it in windows?

Reply:
I took out the testily wrapper so the test should run in windows too.